Quote.go

```diff
@@ -17,6 +17,7 @@ type Quote struct {
 	AnimeID       string    `json:"animeId" editable:"true"`
 	EpisodeNumber int       `json:"episode" editable:"true"`
 	Time          int       `json:"time" editable:"true"`
+	IsMainQuote   bool      `json:"isMainQuote" editable:"true"`
```

Comment:
This field is [redundant](https://en.wikipedia.org/wiki/Single_source_of_truth). It can be a function returning:

`quote.Character().MainQuoteID == quote.ID`